core/mcpmodel.gms

```diff
@@ -99,9 +99,9 @@ parameter	theta_X_PD(r,g)		Value share (output to PD market),
 		theta_X_PN(r,g)		Value share (output to PN market),
 		theta_X_PFX(r,g)	Value share (output to PFX market);
 
-$echo	theta_X_PD(r,g) = xd0(r,g)/s0(r,g);		>>%gams.scrdir%MCPMODEL.GEN	
-$echo	theta_X_PN(r,g) = xn0(r,g)/s0(r,g);		>>%gams.scrdir%MCPMODEL.GEN	
-$echo	theta_X_PFX(r,g) = (x0(r,g)-rx0(r,g))/s0(r,g);	>>%gams.scrdir%MCPMODEL.GEN	
+$echo	theta_X_PD(r,g)$s0(r,g) = xd0(r,g)/s0(r,g);		>>%gams.scrdir%MCPMODEL.GEN	
+$echo	theta_X_PN(r,g)$s0(r,g) = xn0(r,g)/s0(r,g);		>>%gams.scrdir%MCPMODEL.GEN	
+$echo	theta_X_PFX(r,g)$s0(r,g) = (x0(r,g)-rx0(r,g))/s0(r,g);	>>%gams.scrdir%MCPMODEL.GEN	

```

Comment:
There are 5 zero values in s0, as seen below. The code `smn` is "Support activities for mining (213)".

![image](https://github.com/uw-windc/windc_build/assets/19531741/78c5821c-484c-4485-b7c4-aafa75cfa289)

Reply:
The smn sector is a weird one for DC. It comes from being a byproduct of the mch sector (DC doesn't have any mining going on...). I found that removing tiny numbers helps and everything still solves.